packages/react/documentation/coding-conventions.md

```diff
@@ -0,0 +1,31 @@
+# Coding Conventions
+
+## Barrel files
+
+In our React package we use barrel files (`index.ts` in the root and in every component folder) to simplify the import of React components.
+Barrel files allow a consumer to do this:
+
+`import { Heading, Link, Paragraph } from '@amsterdam/design-system-react'`
+
+Instead of this:
```

Comment:
Oh wait, I hadn’t realised the `dist` directory being necessary below. We could highlight the undesirability of that approach:

```suggestion
Instead of a separate import for each component, while even reaching into the `dist` directory:
```